docs/conf.py

```diff
@@ -244,6 +244,8 @@
 # sphinx-codeautolink
 
 codeautolink_concat_default = True
+codeautolink_warn_on_failed_resolve = False  # turn on for debugging
+codeautolink_warn_on_missing_inventory = False  # turn on for debugging
```

Comment:
🏕️ These are useful flags that can help us figure out problems with `sphinx-codeautolink`. The default is `False` for these, but I'm putting these here for convenience. Later on, we could set these to `True` and fix the broken autolinks.